extensions/blocks/image-compare/image-compare.php

```diff
@@ -49,12 +49,21 @@ function load_assets( $attr, $content ) {
 /**
  * Render image compare block for AMP
  *
- * @param array $attr Array containing the image-compare block attributes.
+ * @param array  $attr    Array containing the image-compare block attributes.
+ * @param string $content String containing the image-compare block content.
  *
  * @return string
  */
-function render_amp( $attr ) {
-	// @TODO: what to show for AMP view?
-	// :shrug-emoji:
-	return '';
+function render_amp( $attr, $content ) {
+	// Attributes that are parsed in JavaScript are not passed in, so to get
+	// the URLs we need to parse the content in PHP, which is rather lame.
+	preg_match_all( '/src="([^"]+)"/', $content, $imgs );
+	if ( empty( $imgs ) || ( count( $imgs ) < 2 ) || empty( $imgs[1] ) || ( count( $imgs[1] ) < 2 ) ) {
+		return '';
+	}
+	return sprintf(
+		'<amp-image-slider layout="responsive" width="300" height="200"><amp-img src="%s" layout="fill"></amp-img><amp-img src="%s" layout="fill"></amp-img></amp-image-slider>',
+		$imgs[1][0],
+		$imgs[1][1]
```

Comment:
I would recommend escaping those as we output them.

Reply:
:+1: